converters/fromZigbee.js

```diff
@@ -730,6 +809,47 @@ const converters = {
             return payload;
         },
     },
+    lixee_haElectricalMeasurement: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+
+            const elements = [
+                /* 0x0305 */ 'totalReactivePower',
+                /* 0x0505 */ 'rmsVoltage',
+                /* 0x0508 */ 'rmsCurrent',
+                /* 0x050A */ 'rmsCurrentMax',
+                /* 0x050B */ 'activePower',
+                /* 0x050D */ 'activePowerMax',
+                /* 0x050E */ 'reactivePower',
+                /* 0x050F */ 'apparentPower',
+                /* 0x0511 */ 'averageRmsVoltageMeasPeriod',
+                /* 0x0905 */ 'rmsVoltagePhB',
+                /* 0x0908 */ 'rmsCurrentPhB',
+                /* 0x090A */ 'rmsCurrentMaxPhB',
+                /* 0x090B */ 'activePowerPhB',
+                /* 0x090E */ 'reactivePowerPhB',
+                /* 0x090D */ 'activePowerMaxPhB',
+                /* 0x090F */ 'apparentPowerPhB',
+                /* 0x0911 */ 'averageRmsVoltageMeasurePeriodPhB',
+                /* 0x0A05 */ 'rmsVoltagePhC',
+                /* 0x0A08 */ 'rmsCurrentPhC',
+                /* 0x0A0A */ 'rmsCurrentMaxPhC',
+                /* 0x0A0D */ 'activePowerMaxPhC',
+                /* 0x0A0E */ 'reactivePowerPhC',
+                /* 0x0A0F */ 'apparentPowerPhC',
+                /* 0x0A11 */ 'averageRmsVoltageMeasPeriodPhC',
+            ];
+
+            for (const at of elements) {
+                if (msg.data[at]) {
+                    result[at] = msg.data[at];
+                }
+            }
+            return result;
+        },
+    },
```

Comment:
Can you move all added converter to `lixee.js`? (with `fzLocal` like in the external converters example)